mirdata/datasets/scms.py

```diff
@@ -0,0 +1,302 @@
+        EventData: vocal activations
+    """
+    intervals = []
+    events = []
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        intervals.append([float(line[0]), float(line[1])])
+        events.append(line[2].replace(" ", ""))
+
+    # if file is empty, return None
+    if len(intervals) == 0:
+        return None
+
+    return annotations.EventData(np.array(intervals), "s", events, "open")
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """
+    The Saraga-Carnatic-Melody-Synth dataset
+    """
+
+    def __init__(self, data_home=None, version="default"):
+        super().__init__(
+            data_home,
+            version,
+            name="scms",
+            track_class=Track,
+            bibtex=BIBTEX,
+            indexes=INDEXES,
+            remotes=REMOTES,
+            download_info=DOWNLOAD_INFO,
+            license_info=LICENSE_INFO,
+        )
+
+    @core.cached_property
+    def _artists_to_track_mapping(self):
+        mapping_path = os.path.join(
+            self.data_home, "SCMS/artists_to_track_mapping.json"
+        )
+
+        try:
+            with open(mapping_path, "r") as fhandle:
+                mapping = json.load(fhandle)
+        except FileNotFoundError:
+            raise FileNotFoundError(
+                "Artists to track mapping not found. Did you run .download()?"
+            )
+
+        return mapping
+
+    @core.cached_property
+    def _metadata(self):
+        metadata_path = os.path.join(self.data_home, "SCMS/metadata.json")
+        try:
+            with open(metadata_path, "r") as fhandle:
+                artists_metadata = json.load(fhandle)
+        except FileNotFoundError:
+            raise FileNotFoundError("Metadata not found. Did you run .download()?")
+
+        mapping = self._artists_to_track_mapping
+        idxs_and_artists = []
+        for artist in list(mapping.keys()):
+            for track in mapping[artist]:
+                idxs_and_artists.append((track, artist))
+
+        artist_info = {}
+        for subset in list(artists_metadata.keys()):
+            for artist, info in artists_metadata[subset].items():
+                artist_info[artist] = {
+                    "tonic": info["tonic"],
+                    "gender": info["gender"],
+                    "train": True if subset == "train" else False,
+                }
+
+        metadata = {}
+        for idx in idxs_and_artists:
+            metadata[idx[0]] = {
+                "artist": idx[1],
+                "title": " ".join(idx[0].split("_")[:-1]),
+                "tonic": artist_info[idx[1]]["tonic"],
+                "gender": artist_info[idx[1]]["gender"],
+                "train": artist_info[idx[1]]["train"],
+            }
+
+        return metadata
+
+    def load_audio(self, *args, **kwargs):
+        return load_audio(*args, **kwargs)
+
+    def load_pitch(self, *args, **kwargs):
+        return load_pitch(*args, **kwargs)
```

Comment:
Same here, is this the issue we discussed today that appears in most loaders?

Reply:
missed testing `load_audio` actually (done it now), but `load_pitch` and `load_activations` are actually tested, so I wonder if yeah, is one of these testing problems we discussed...

Reply:
ok sorry @magdalenafuentes, these functions should not be there, were deprecated a few versions ago. The newest loaders do not have them actually, started from the wrong template hehe. Fixed!